README.md

```diff
@@ -12,6 +13,11 @@
 1. Once you have an SSH key, clone the [sc1-data-format repository](https://github.com/badgerloop-software/sc1-data-format) to your computer. Make sure to clone it using SSH (when you go to copy the clone link, there will be an SSH option above the link).
 2. Next, `cd` into the `chase-car-dashboard` repository and run `git submodule update --init`.
 
+## Updating the Submodule
+
+0. This step assumes that the repository has been initialized as above.
+1. Run the command `npm run update-data`. This should take care of updating the submodule and JSON files in the frontend. (If the command fails to run, make sure you have the most recent dependencies and version of `npm` installed onto your computer).
```

Comment:
running `npm run update-data` does not only update the submodule. There should be more details regarding what specifically this does. In addition, there should be a way to run the update/convert data scripts without updating the submodule (though it should still be made clear that you should update the submodule first) in case someone cannot use git in the same terminal that they use to run the dashboard. At the very least, this could mention what scripts are run using this general script (which the user could instead run individually) and what each of them do

Reply:
Also, there are steps below in "Running the Dashboard" and "Contributing to the Dashboard" that mention updating the submodule and data constants. Those steps should be replaced with references to this section.

Reply:
Updated `README` to reflect updates in the code, as requested